guides/advanced_guides/configuration.md

```diff
@@ -287,39 +295,43 @@ This command will stop the process if:
 
 If this command is not called, no snapshot will be imported.
 
-### Ignore missing snapshot
+### Ignore Missing Snapshot
 
 **Environment variable**: `MEILI_IGNORE_MISSING_SNAPSHOT`
 **CLI option**: `--ignore-missing-snapshot`
 
 The engine ignores missing snapshots and does not throw an error in this case.
 
-### Ignore snapshot if db exists
+Requires `--import-snapshot` to be defined.
+
+### Ignore Snapshot If DB Exists
 
 **Environment variable**: `MEILI_IGNORE_SNAPSHOT_IF_DB_EXISTS`
 **CLI option**: `--ignore-snapshot-if-db-exists`
 
 The engine skips snapshot importation if a database already exists. No error is thrown in this case.
```

Comment:
```suggestion
The engine skips importing the given snapshot if a database already exists, and instead loads the existing database. No error is thrown in this case.
```

Reply:
Sorry @react-learner , English is probably better but it's harder for me to understand your version. I needed to read it 4 times to understand it. Have you another sentence to submit?

Reply:
@LegendreM You're right, it's confusing. How about this instead?

```suggestion
If a database already exists, MeiliSearch will attempt to launch using that database instead of importing a snapshot. No error is thrown in this case.
```